content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -351,3 +458,37 @@ The label to instruct Istio to automatically include applications in the `defaul
 {{< text bash >}}
 $ kubectl label namespace default istio.io/dataplane-mode-
 {{< /text >}}
+
+{{< tabset category-name="config-api" >}}
+
+{{< tab name="Istio classic" category-value="istio-classic" >}}
+
+To remove the `bookinfo` Gateway:
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl delete gw/bookinfo-gateway
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+To remove the `reviews` Service subsets and HTTPRoute:
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl delete -f @samples/bookinfo/platform/kube/bookinfo-versions.yaml@
+{{< /text >}}
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl delete -f httproute/reviews
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+If you installed the Gateway API CRDs, remove them:
+
+{{< text syntax=bash snip_id=none >}}
```

Comment:
You should call this in the not gateway-api cleanup.
```suggestion
{{< text syntax=bash snip_id=delete_crds >}}
```

Reply:
As suggested in a different thread of this PR, the doc uses the gateway api [boilerplate](https://github.com/istio/istio.io/blob/master/content/en/boilerplates/gateway-api-support.md). Because of this, the test sources `tests/util/gateway-api.sh` to install the CRDs by calling `install_gateway_api_crds` and remove the CRDs by calling `remove_gateway_api_crds`.